packages/org-unit-dialog/src/OrgUnitDialog.js

```diff
@@ -116,6 +117,12 @@ OrgUnitDialog.propTypes = {
     */
     handleUserOrgUnitClick: PropTypes.func.isRequired,
 
+    /**
+     * Callback handler for selecting user orgunit
+     * Arguments supplied in callback: event, checked
+     */
+    handleMultipleOrgUnitsSelect: PropTypes.func.isRequired,
```

Comment:
So this will be a breaking change then.

Reply:
I can make it optional, just wondering if it causes problems?